docs/src/main/paradox/errorhandling.md

```diff
@@ -20,6 +20,11 @@ Scala
 Java
 : @@ snip [stopping](../../test/java/sample/javadsl/StreamWrapperActor.java) { #errorHandlingStop }
 
+Stopping the actor on stream failure in the [Stream-ClosedGraph](http://doc.akka.io/docs/akka/snapshot/scala/stream/stream-graphs.html) scenario
```

Comment:
* The link could be a relative `@ref` link to the relevant `.md` file, that enables link checking if we ever move that page.
* I feel this still deserves some explanation: to a new reader "the Stream-ClosedGraph scenario" doesn't mean much, and the linked page doesn't really clarify it either. I think it would be good to spend a sentence or two explaining what scenario this example is about.